shared/router-v2/router.shared.js

```diff
@@ -32,17 +32,27 @@ export const oldActionToNewActions = (action: any, navigation: any) => {
       const p = action.payload.path.last
         ? action.payload.path.last()
         : action.payload.path[action.payload.path.length - 1]
-      if (!p) {
+      // We have to check the parent path as well, since sometimes we navigate
```

Comment:
I see these on `navigateTo`, I thought `navigateAppend` typically has the `path` set

Reply:
In any case I'm not sure this change should be made, since most other nav2 things are done & were tested with the existing behavior

Reply:
`navigateTo` falls through to this

Reply:
yeah, I'm saying I don't think `parentPath` should be used in new routing calculations